Frida/RpcRequest.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+internal enum RpcInternalResult {
+    case success(value: Any?)
+    case error(error: String)
+}
+
+public enum RpcResult<T> {
+    case success(T)
+    case error(Swift.Error)
+}
```

Comment:
These two look like lower level than `RpcRequest` and should go below it. `RpcResult` should probably go first of the two, as it's higher level.